docs/11_0_0/components/outputpanel.md

```diff
@@ -33,6 +33,7 @@ styleClass | null | String | StyleClass of the HTML container element
 layout | block | String | Shortcut for the css display property, valid values are block(default) and inline.
 deferred | false | Boolean | Deferred mode loads the contents after page load to speed up page load.
 deferredMode | load | String | Defines deferred loading mode, valid values are "load" (after page load) and "visible" (once the panel is visible on scroll).
+loadNeeded | false | Boolean | Indicates deferred load is needed.
```

Comment:
Hmmm not sure about `loadNeeded` name.  Maybe `deferredCheck` or `deferredEnabled` or `deferOnAjax`?  I will let @tandraschko chime in.